utils.py

```diff
@@ -7,15 +7,15 @@
 import requests
 
 from models import Game, Player, session
-from vars import EMOJI, TIMEZONE_CET, TIMEZONE_UTC
+from vars import EMOJI, TIMEZONE_CET, TIMEZONE_UTC, CSGO_NICKNAMES
 
 
 def sync_player_data(player: Player, user):
     """Updates player's data if it has changed"""
-    p_data = [player.username, player.first_name, player.last_name]
-    u_data = [user.username, user.first_name, user.last_name]
+    p_data = [player.username, player.first_name, player.last_name, player.csgo_nickname]
+    u_data = [user.username, user.first_name, user.last_name, user.csgo_nickname]
     if p_data != u_data:
-        player.username, player.first_name, player.last_name = u_data
+        player.username, player.first_name, player.last_name, player.csgo_nickname = u_data
         player.save()
 
```

Comment:
Changes here are not needed because it's compares Telegram users' data with Players' data.
Thus, Telegram user don't have `csgo_nickname` property.

Fosctring here is not explicit enough (my bad).

Reply:
*Docstring

Reply:
Actually we need to sync `player.scgo_nickname` with `CSGO_NICKNAMES[user.username]` here I guess

Reply:
> Changes here are not needed because it's compares Telegram users' data with Players' data.
> Thus, Telegram user don't have `csgo_nickname` property.
> 
> Fosctring here is not explicit enough (my bad).

Removed.